docs/Using-Fleet/configuration-files/README.md

```diff
@@ -420,109 +1339,3 @@ spec:
           - /Users/wxs/sigs/baz.sig
     overrides: {}
 ```
-
```

Comment:
All this deleted section has been integrated more consistently in the corresponding section of the appconfig settings.

Reply:
Thanks for the clean up.